src/main/java/edu/uc/inventorymanager/ui/MainController.java

```diff
@@ -68,8 +68,7 @@ public ResponseEntity fetchItemById(@PathVariable("id") int id) {
         var item = itemService.fetchItemById(id);
         if (item == null) {
             return new ResponseEntity(HttpStatus.NOT_FOUND);
-        }
-        return new ResponseEntity(HttpStatus.OK);
+        } else return new ResponseEntity(HttpStatus.OK);
```

Comment:
I forgot the else statement in my last commit...

Reply:
Surround with curlies as well, to keep it consistent.

Reply:
The else statement here is unnecessary because if the condition is met, it will return and end the function and if it is not, it will continue down to the next return